site/app/views/forum/ForumThreadView.php

```diff
@@ -283,6 +299,8 @@ function changeName(element, user, visible_username, anon){
 							{$onChange}
```

Comment:
If after using filter no threads are left, then if we reload page(keeping same filter).
`$onChange` remains empty string. Thus, we are not able to change filter.

```
$onChange = '';
if($thread_count > 0) {
 		$onChange = <<<HTML
 		modifyThreadList({$currentThread}, '{$currentCategoriesIds_string}');
HTML;
}
```